src/Cake.Issues.DupFinder/DupFinderIssuesProvider.cs

```diff
@@ -79,31 +84,58 @@ private static string GetIdentifier(int cost, IReadOnlyCollection<DuplicateFragm
             return $"{fragment.FilePath}-{cost}-{string.Join("-", otherFragments)}";
         }
 
-        private static string GetHtmlMessage(int cost, IReadOnlyCollection<DuplicateFragment> fragments, DuplicateFragment fragment)
+        private void AddFileLinks(IEnumerable<IIssue> result)
```

Comment:
You don't need to set `IIssue.FileLink) here in the provider as it already will be set by the issue reader

Reply:
Where exactly can I see this?

Reply:
Here: https://github.com/cake-contrib/Cake.Issues/blob/release/0.9.0/src/Cake.Issues/IssuesReader.cs#L68